corehq/apps/locations/static/locations/js/location_types.js

```diff
@@ -321,6 +321,41 @@ hqDefine('locations/js/location_types.js', function(){
             };
         };
     }
+
+    $(function() {
```

Comment:
OK, great! It sounds like you're keeping the right stuff in mind as you're doing this. No need (for me) to discuss more now.